modules/administration/pages/space-management.adoc

```diff
@@ -0,0 +1,67 @@
+[[space-management]]
+= Managing Disk Space
+
+Running out of disk space can have a severe impact on the {productname} database and file structure which, in most cases, is not recoverable.
+
+{productname} monitors some directories for free disk space.
+You can modify which directories are monitored, and the warnings that are created.
+All settings are configured in the [path]``/etc/rhn/rhn.conf`` configuration file.
+
+
+
+== Monitored Directories
+
+By default, {productname} monitors these directories:
+
+* [path]``/var/lib/pgsql``
+* [path]``/var/spacewalk``
+* [path]``/var/cache /srv``
+
+You can change which directories are monitored with the [systemitem]``spacecheck_dirs`` parameter.
+You can specify multiple directories by separating them with a space.
+
+For example:
+
+----
+spacecheck_dirs = /var/lib/pgsql var/spacewalk /var/cache /srv
+----
+
+
+
+== Thresholds
+
+By default, {productname} will create a warning email when a monitored directory has less than 10% of total space available.
```

Comment:
Just spotting...  we use "email" and "e-mail" (the latter per SUSE style guide, iirc).  I'd rather vote to go for just "mail" :)

Reply:
hrm, IBM says "email". I think I'm with you on erring on the side of "mail", though ;)